pyhgvs/utils.py

```diff
@@ -2,7 +2,11 @@
 Helper functions.
 """
 
-from itertools import imap
+try:
+    from itertools import imap
+except ImportError:
+    # Python 3
+    imap=map
```

Comment:
@lucaswiman Is this OK with you or would you prefer `from six.moves import map as imap`?

Reply:
No reason to add `six` if this is the only place it's needed. I'd prefer to have python 3 be the "default" python and, like:
```python
try:
     from itertools import imap as map
except ImportError:  # python 3
    pass

...

for trans in map(make_transcript, read_refgene(refgene_file)):
```